channel/channel.go

```diff
@@ -251,6 +256,12 @@ func (c *Channel) AddSignedState(ss state.SignedState) bool {
 		return false
 	}
 
+	if prevSs, err := c.LatestSignedState(); err != nil {
```

Comment:
prevSignedState ?
auditors will be rather strict on naming things, especially signatures.

Reply:
this will be removed, it will be managed by specific objectives